cmd/internal/applicationsnapshot/source.go

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright © 2022 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package applicationsnapshot
+
+import (
+	"context"
+	"io/ioutil"
+	"os"
+
+	"github.com/hashicorp/go-getter"
+)
+
+const SCHEMA = "https://raw.githubusercontent.com/redhat-appstudio/release-service/main/config/crd/bases/appstudio.redhat.com_applicationsnapshots.yaml"
```

Comment:
I think the getter client can be forced here, so `https` works out of the box below to make the invocation a bit simpler.

```suggestion
const SCHEMA = "http::https://raw.githubusercontent.com/redhat-appstudio/release-service/main/config/crd/bases/appstudio.redhat.com_applicationsnapshots.yaml"
```

Reply:
Weirdly, without the config I get ` no source URL was returned`